### DISCUSSION TITLE
Flows: Can I use flow object variables in a trigger?

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15798] [Archive]
@directus/team 
I noticed some inconsistencies in the syntax to access flow object variables across triggers and operations. 

For example:
1) In any trigger or operation that uses `Response Body`, we don't use `{{ }}` syntax.
2) In most operations, we can type `{{ opKey }}` with spaces.

<img width="603" alt="Screen Shot 2022-09-28 at 4 54 24 PM" src="https://user-images.githubusercontent.com/62306630/192895710-92adf5bf-a41a-4e8f-9bda-93abf5bbc4ca.png">

---

<img width="831" alt="Screen Shot 2022-09-28 at 4 54 54 PM" src="https://user-images.githubusercontent.com/62306630/192895759-20b62fc6-a8f8-4f5a-ab83-fd661d4fe413.png">

**Comment by licitdev** [comment]
IIRC, the `Response Body` field is only located in the Trigger Options, where Trigger Options do not support variables. 
Whereas, values in Operation Options can utilise the `{{ }}` syntax for access to variables.
